server/fleet/packs.go

```diff
@@ -16,8 +16,11 @@ type Pack struct {
 	Platform    string  `json:"platform,omitempty"`
 	Disabled    bool    `json:"disabled"`
 	Type        *string `json:"type" db:"pack_type"`
+	Labels      []Label `json:"labels"`
```

Comment:
Maybe define a new struct with only the things you need to return:
```
type LabelIdentifier struct {
  ID uint `json:"id"`
  Name string `json:"name"`
}
```
(same for the other two). 

Otherwise it will always have all those other values of `Label` empty.